tin-can-phone.ino

```diff
@@ -401,12 +403,8 @@ void readSerial () {
 
 void setup() {
     USE_SERIAL.begin(115200);
-    //can.connectToWifi("Recurse Center", "nevergraduate!");
-    //can.connectToWifi("MySpectrumWiFib8-2G", "classypoodle861");
-    can.connectToWifi("s&m", "passwordispassword");
-    //can.openWebSocket("10.0.20.109", 3000); Recurse
-    //can.openWebSocket("192.168.1.18", 3000); //David's House
-    can.openWebSocket("192.168.1.2", 3000); //Mikey's House
+    can.connectToWifi(TIN_CAN_SSID, TIN_CAN_PASSWORD);
+    can.openWebSocket(TIN_CAN_SWITCH, 3000);
```

Comment:
Hm, I forgot about the port. Let's add another configuration constant for that and change the name of `TIN_CAN_SWITCH` to represent the host (i.e. server name or ip). In other words, we'd have:

- `TIN_CAN_SWITCH_HOST`
- `TIN_CAN_SWITCH_PORT`